assets/src/edit-story/elements/video/output.js

```diff
@@ -42,11 +42,23 @@ function VideoOutput({ element, box }) {
     loop: loop ? 'loop' : undefined,
   };
 
+  const tracks = defaultForUndefined(element.tracks, resource.tracks);
```

Comment:
`resource.tracks` is not a thing? Tracks are part of `element` itself, not `resource`. So this line seems incorrect.

Reply:
It was in the original version. I will fix.

Reply:
Fixed.